docs/src/Style.ts

```diff
@@ -37,4 +37,11 @@ export const semanticCssOverrides = `
     color: rgba(255,255,255,.7)!important;
     cursor: not-allowed!important;
   }
+
+  button:-moz-focusring,
+  [type="button"]:-moz-focusring,
+  [type="reset"]:-moz-focusring,
+  [type="submit"]:-moz-focusring {
+    outline: 0;
+  }
```

Comment:
This will only work in the doc site, is that the intent?  If this should be the style for all usages of the Teams theme, including outside of the doc site by consumers, then this should be a `staticStyle` in the Teams theme.  This way, everywhere the Teams theme is used, there is no outline in Firefox.

Reply:
for now yes, the intent was just to address doc site issue and propose a question of whether the same change should be applied for Teams theme.